packages/ui-components/src/components/CheckboxJs/Checkbox.stories.js

```diff
@@ -6,7 +6,7 @@
 import { Checkbox } from "./index.js"
 
 export default {
-  title: "Forms/Checkbox/Checkbox",
+  title: "Forms/Checkbox/Checkbox (JS, deprecated)",
```

Comment:
Please move deprecated components to the deprecated section in storybook so we don't clutter up the documentation.
```suggestion
  title: "Deprecated/Checkbox/Checkbox (JS, deprecated)",
```

Reply:
Done, in next PR I'll do reorganization of migrated / interim / deprected components, so I'll update this in other places too.